### PR TITLE
Switch CI to running on 1ES

### PR DESCRIPTION
**Pull request by hawkowl**
### Which issue this PR addresses:

Fixes https://issues.redhat.com/browse/ARO-1401 maybe

### CODE REVIEW

**Review by cblecker — APPROVED** [review]

File: .pipelines/ci.yml [hunk 1]
```diff
@@ -18,49 +18,73 @@ pr:
     exclude:
     - docs/*
 
+resources:
+  containers:
+  - container: golang
+    image: registry.access.redhat.com/ubi8/go-toolset:1.17
```
Comment (weinong) [inline .pipelines/ci.yml:24]: when will we upgrade to 1.18?

**Review by weinong — APPROVED** [review]
LGTM

File: .pipelines/ci.yml (hunk 1 above)
Comment (hawkowl) [reply .pipelines/ci.yml:24]: When OpenShift 4.11 happens, I believe.

**Comment by darthhexx** [comment]
Wanted to highlight that once this is merged we will be effectively disabling all Admin Portal E2E tests. This still needs to be addressed as part of the migration.

**Comment by hawkowl** [comment]
@darthhexx E2E migration is not a part of this ticket -- it will be covered :)

**Pull request merged**